code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -195,10 +203,7 @@ GLOBAL_VAR_INIT(cryo_overlay_cover_off, mutable_appearance('icons/obj/cryogenics
 	. = on
 	on = new_value
 	update_icon()
-	if(on)
-		occupant_vis.on_toggle_on()
-	else
-		occupant_vis.on_toggle_off()
+	SEND_SIGNAL(src, COMSIG_CRYO_SET_ON, new_value)
```

Comment:
Could you either add the signal before the `on` value changes, or change its name to reflect it happens after, while also passing the previous value of `on`?

Reply:
Signal now happens before